retail/3.2/modules/ROOT/pages/rcg-sfdc-customization-guide.adoc

```diff
@@ -290,7 +290,7 @@ Unique fields should be case-sensitive.
 
 === Create Fields via Apex Script
 
-Use the Apex class `CustomFieldUtility` to create custom fields programmatically. See the https://anypoint.mulesoft.com/exchange/org.mule.examples/rcg-common-resources/[RCG Common Resources] project to find the source for this class. Once you have downloaded the project, follow these steps:
+Use the Apex class `CustomFieldUtility` to create custom fields programmatically. See the https://anypoint.mulesoft.com/exchange/org.mule.examples/rcg-common-resources-src/[RCG Common Resources] project to find the source for this class. Once you have downloaded the project, follow these steps:
```

Comment:
```suggestion
Use the Apex class `CustomFieldUtility` to create custom fields programmatically. See the https://anypoint.mulesoft.com/exchange/org.mule.examples/rcg-common-resources-src/[RCG Common Resources] project to find the source for this class. Download the project and follow these steps:
```

Reply:
This is one of those changes that needs to be back-ported to beta.